feature-group-definitions/show-picker-input.dist.yml

```diff
@@ -0,0 +1,24 @@
+# Generated from: show-picker-input.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: showPicker() for <input>
+description: The `showPicker()` method for `<input>` elements shows the user interface for picking a value. For example, for `<input type="date">` it shows the interface for picking a date.
+spec: https://html.spec.whatwg.org/multipage/input.html#dom-input-showpicker
+usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/4097
+status:
+  baseline: false
```

Comment:
This should be Baseline, it looks like a usable subset is supported:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/showPicker#browser_compatibility